go/base/static/js/src/apps/dialogue/models.js

```diff
@@ -182,6 +183,39 @@
       }).defaults(GroupStateModel.__super__.defaults.call(this));
     }
   });
+  
+  var HttpJsonStateModel = DialogueStateModel.extend({
+    storableOnContact: false,
+
+    relations: [{
+      type: Backbone.HasOne,
+      key: 'url',
+      includeInJSON: 'url'
+    }, {
+      type: Backbone.HasOne,
+      key: 'method',
+      includeInJSON: 'method'
+    }, {
+      type: Backbone.HasOne,
+      key: 'entry_endpoint',
+      relatedModel: DialogueEndpointModel
+    }, {
+      type: Backbone.HasOne,
+      key: 'exit_endpoint',
+      relatedModel: DialogueEndpointModel
+    }],
```

Comment:
I'm not sure if backbone-relational allows relations that are actually not backbone model types, but I think we should try stick to only having relations for backbone model types (so `url` and `method` would go).